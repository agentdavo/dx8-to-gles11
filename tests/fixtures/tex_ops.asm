tex t0
texld t1, t0
texcrd r0, t1
texkill t2
sub r0, v0, v1
