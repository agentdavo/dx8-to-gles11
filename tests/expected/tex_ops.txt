TEX_SAMPLE stage=0
TEX_LOAD stage=1
TEX_COORD_COPY
TEX_KILL
TEX_ENV_COMBINE func=34023
